data/update.py

```diff
@@ -63,14 +64,23 @@ def update(scan_mode, gather_mode, options):
         elif gather_mode == "skip":
             LOGGER.info("Skipping subdomain gathering.")
 
+        # One day, these variables puled globally from env.py will be gone
```

Comment:
please don't hate me, but "pulled" :P

Reply:
I was just fixing that when you commented on it